services/api/pyproject.toml

```diff
@@ -54,3 +56,12 @@ preview = true
 
 [tool.mypy]
 strict = true
+
+
+[[tool.mypy.overrides]]
+module = "hffs.*"
```

Comment:
Looks like multiple packages can be added in one list https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml-file
```
[[tool.mypy.overrides]]
module = [
    'packagename',
    'packagename2'
]
...
```
But not sure if it applies in this case

Reply:
done in https://github.com/huggingface/datasets-server/pull/875/commits/718448634c81d031e8d552376b713f29a99772f0, thanks